src/ConfigEditor.tsx

```diff
@@ -55,6 +70,64 @@ export class ConfigEditor extends PureComponent<Props, State> {
                   width={40}
                   onReset={onResetToken}
                 />
+              </InlineField>
+            </div>
+          </div>
+
+          <h3 className="page-heading">Client Credential Trino Authentication</h3>
```

Comment:
OAuth might be more familiar to users and `client credential` is one of the variants of OAuth, right?
```suggestion
          <h3 className="page-heading">OAuth Trino Authentication</h3>
```